pkg/util/ipvs/kernelcheck_linux_test.go

```diff
@@ -30,65 +31,74 @@ func TestRequiredIPVSKernelModulesAvailableCheck(t *testing.T) {
 		loadedKernel  string
 		kernelVersion string
 		builtinKernel string
+		modinfoError  error
 
 		expectErrors   bool
 		expectWarnings bool
 	}{
 		{
-			caseName:       "no installed kernel modules and no builtin kernel modules",
+			caseName:       "no loaded, no installed and no builtin kernel modules",
 			loadedKernel:   "",
 			kernelVersion:  "3.13.0-24-generic",
 			builtinKernel:  "",
+			modinfoError:   fmt.Errorf("modinfo error"),
 			expectErrors:   false,
```

Comment:
why `modinfoError` not nil?

Reply:
I think I already answered this. Answering again:
It's not nil as this test case expects loadable modules to be installed.